vars/test.yml

```diff
@@ -299,7 +299,7 @@ windows_has_inbox_driver: true
 # 'vmtools_url_path' takes precedence over 'vmtools_iso_path'.
 #
 vmtools_esxi_bundled: false
-vmtools_url_path: "https://packages.vmware.com/tools/releases/12.1.5/windows/VMware-tools-windows-12.1.5-20735119.iso"
+vmtools_url_path: "https://packages.vmware.com/tools/releases/12.2.0/windows/VMware-tools-windows-12.2.0-21223074.iso"
 # vmtools_iso_path: "[datastore1] VMware/VMTools/12.1.5/windows/VMware-tools-windows-12.1.5-20735119.iso"
```

Comment:
This line can also be updated with 12.2.0 iso. By the way, after your modification, you can run command:
```
git commit -s --amend
```
to add your sign-off. And then push your changes:
```
git push origin Winnie81-patch-1:Winnie81-patch-1 -f
```